react/features/toolbox/middleware.js

```diff
@@ -7,6 +7,48 @@ import {
     SET_TOOLBOX_TIMEOUT
 } from './actionTypes';
 
+import {
+    SET_VIDEO_AVAILABLE,
+    SET_VIDEO_MUTED
+} from '../../features/base/media/actionTypes';
+
+import {
+    setToolbarButton
+} from './actions.native.js';
```

Comment:
I want to make sure that this import is correct. I know web toolbox does something odd with importing functions from a .native.js file. Would it not be possible to import without specifying .native.js or .web.js?